scripts/incentive-spreadsheet-to-json.ts

```diff
@@ -53,16 +54,26 @@ async function convertToJson(
     }
   });
 
+  const dir = path.dirname(file.filepath);
+  if (!fs.existsSync(dir)) fs.mkdirSync(dir);
```

Comment:
before, did we document anywhere that a user needed to make a directory? I don't think I saw it in the README.

Reply:
It was mentioned – so now I guess that's unnecessary. I've removed those lines. If the user does create it it's not a problem.